sql/rename.go

```diff
@@ -209,6 +213,8 @@ func (p *planner) RenameIndex(n *parser.RenameIndex) (planNode, error) {
 	if err := p.txn.Put(descKey, tableDesc); err != nil {
 		return nil, err
 	}
+	// Mark transaction as operating on the system DB.
+	p.txn.SetSystemDBTrigger()
```

Comment:
yeah, `maybeGossipConfig` being called pretty much all the time is pretty unfortunate and I think Marc is going the right way with this. :+1: